shared/testutil/altair.go

```diff
@@ -197,16 +197,19 @@ func buildGenesisBeaconState(genesisTime uint64, preState state.BeaconStateAltai
 		BodyRoot:   bodyRoot[:],
 	}
 
-	var pubKeys [][]byte
-	for i := uint64(0); i < params.BeaconConfig().SyncCommitteeSize; i++ {
-		pubKeys = append(pubKeys, bytesutil.PadTo([]byte{}, params.BeaconConfig().BLSPubkeyLength))
+	syncCommittee := make([][]byte, params.BeaconConfig().SyncCommitteeSize)
+	vals := preState.Validators()
+	valsCount := len(vals)
+	for i := 0; i < len(syncCommittee); i++ {
+		// Wrap around in case the number of validators is less than sync committee size.
```

Comment:
impossible condition based on spec, if number of validators is less than sync committee size, we should just error out, rather find a bug than work around